app/views/content/cookies.md

```diff
@@ -0,0 +1,51 @@
+Apply for teacher training puts small files (known as ‘cookies’) onto your computer to collect information about how you use the site.
+
+Cookies are used to:
+
+* measure how you use the this service so it can be updated and improved based on your needs
+* remember the notifications you’ve seen so that we don’t show them to you again
+* remember who you are when you have logged in so we can show you your information and information relevant to you
+* ’Apply for teacher training’ cookies aren’t used to identify you personally.
+
+Find out more about [how to manage cookies](http://www.aboutcookies.org/).
```

Comment:
This feels like a weird thing to link to?

Reply:
This is content @dankmitchell ported from Find. Probably needs reviewing (and potentially across the service line)